src/hooks/useInitialStartup.js

```diff
@@ -168,7 +168,7 @@ export const useInitialStartup = () => {
     LogBox.ignoreAllLogs();
 
     // statusbar
-    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.white);
+    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.almostBlack);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the Android StatusBar background color might affect readability if the status bar content (like time, battery level, etc.) is also in a dark color. Please ensure that this color change doesn't negatively impact the visibility of the status bar content.
```diff
-    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.white);
+    if (Platform.OS === 'android') StatusBar.setBackgroundColor(COLORS.almostBlack);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->